modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} (OCP) update process.
+
+. The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource, which may be managed through the web console or the CLI.
+
+. The Cluster Version Operator (CVO) discovers that the `desiredUpdate` in the ClusterVersion resource differs from the current cluster version.
+Using graph data from the OpenShift Update Service, the CVO resolves the desired cluster version to a pullspec to the release image.
```

Comment:
s/pullspec/pull spec
It seems that is how we spell it. 

Also, I see no references to _pull spec to_. I see a few _pull spec for_ instances. Would that be accurate (it would alleviate the close-proximity double uses of _to_).